.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,61 @@
+name: Bug report
+description: Create a report to help us improve ILGPU
+title: "[BUG]: <title>"
+labels: [ "bug" ]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **Thank you for reporting this issue!**        
+        To help us understand and address the problem effectively, please provide detailed information. 
+        You can refer to [this guide](https://stackoverflow.com/help/mcve) for tips on creating a good bug report.
+
+  - type: textarea
+    id: describe-bug
+    attributes:
+      label: Describe the bug
```

Comment:
ISSUE: Inconsistent capitalization.

- Describe the bug
- Steps To Reproduce
- Expected Behavior
- etc

We should be consistent.